waveform.go

```diff
@@ -48,7 +49,7 @@ func plotAudio(imgfile string, audio []int16, samplerate int) {
 
 	for i, sample := range audio {
 		timestamp := float32(i) / sampleratePerMili
-		f.WriteString(fmt.Sprintf("%f %d %d\n", timestamp, sample))
+		f.WriteString(fmt.Sprintf("%f %d %d\n", timestamp, sample)) // missing format value here
```

Comment:
my linter got this. I was unable to fix  :disappointed:

Reply:
Not sure I understand the problem =(

Reply:
Sprintf expects 3 formats but only 2 was given

Reply:
Remove...one of them ? =P